driver/pelican/pelican.go

```diff
@@ -169,6 +190,50 @@ func (pel *Pelican) GetStatus() (*PelicanStatus, error) {
 		}
 	}
 
+	// Thermostat History Object Request to retrieve time stamps from past hour
```

Comment:
Didn't we say the reporting period for a thermostat was about 10 minutes? If so, I would think we only need to look about 20 minutes into the past.

The farther back we look, the longer the driver will wait before considering the thermostat unreachable and stopping status publication.

@gtfierro Any thoughts on this?

Reply:
Just to offer some extra insight, I just wrote a couple lines of Go that print out the list of timestamps from the past five hours, retrieved from the Thermostat History Object response. I ran this code at 10:30 p.m. From the results published below, I saw that the last push of data was at 21:52, or 9:52 p.m., and it seems that up until then, the thermostat has been updating in around 5-6 minute intervals.

I've seen this behavior before, where there seems to be a thirty minute gap between the current time and the last time stamp. I'm not sure what is accounting for this delay, but my hunch is that it's some latency from the pelican's information getting stored into the web api server. That's why I feel like an hour would be a more appropriate suggestion.

I included a screenshot below here, where I just ran the test at 11:45 p.m. Again, the last push is 11:17 p.m., which is why I feel like the hour time frame works pretty well. However, I'm open to suggestions 👍 

<img width="879" alt="screen shot 2018-04-04 at 11 53 33 pm" src="https://user-images.githubusercontent.com/10174390/38351230-6dec25e6-3863-11e8-993e-d323a4b4859e.png">

Reply:
Okay, I think we're fine with 1 hour then. Just wanted to be sure we could justify our decision on how far back to look.

Reply:
Did we double check what time the thermostat thinks it is? The time synchronization may be off. In any case, 1 hour does sound fine